### ISSUE TITLE
Dock disappeared over night

### ISSUE THREAD

**Issue by JaapM1983** [icedman/dash2dock-lite#83]
I love this extension, especially with a new set of icons. Unfortunately the whole dock disappeared visually somehow. Any thoughts?
[Schermopname van 2024-02-08 08-49-15.webm](https://github.com/icedman/dash2dock-lite/assets/159419116/000dfb43-ee66-45f9-849c-64f27562c14f)

**Comment by icedman** [comment]
Could you tell me your Gnome version. And OS.

**Comment by JaapM1983** [comment]
Ha, thank you! 
Fedora Linux 39 (Workstation Edition)
Gnome 45.3

Have been stressing yesterday, because I switched from X11 to Wayland (with a NVIDIA GeForce RTX™ 3060 Laptop GPU). Then the dock disappeared, so I presumed it was because of the switch to Wayland. Switched back, still no dock. Then suddenly my Nvidia-card wasn't the primary GPU, so my second screen was unavailable. Have been updating drivers, kernel and stuff. Added nvidia.conf with primaryGPU = yes. Still didn't work. Added  section Serverlayout, with AllowNvidiaGPUScreens. Then all of the sudden Nvidia was back the Primary GPU and stuff works like before. 

This morning I installed dash2dock lite again, unfortunately with the same result (so maybe it wasn't the switch to Wayland, but something totally different). 

Now I'm back to using the vanilla-dock, but still hope I get the dash2dock working!
